components/console/introduction.rst

```diff
@@ -124,6 +115,14 @@ This prints::
 Coloring the Output
 ~~~~~~~~~~~~~~~~~~~
 
+.. note::
+
+    By default, the Windows command console doesn't support output coloring. The
+    Console component disables output coloring for Windows systems, but if your
+    commands invoke other scripts which emit color sequences, they will be
+    wrongly displayed as raw escape characters. Install the free `ANSICON`_
```

Comment:
Thanks for your comment. I've added ConEmu as the first recommended application.